sdk/keyvault/Azure.Security.KeyVault.Keys/src/JsonWebKeyConverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Azure.Core;
+
+namespace Azure.Security.KeyVault.Keys
+{
+    /// <summary>
+    /// Converts a <see cref="JsonWebKey"/> to or from JSON.
+    /// </summary>
+#pragma warning disable AZC0014 // Avoid using banned types in public API
+    public class JsonWebKeyConverter : JsonConverter<JsonWebKey>
```

Comment:
Can we make this type internal?

Reply:
Can we? I thought they had to be public. Checking...

Reply:
I see [`GeoJsonConverter`](https://github.com/Azure/azure-sdk-for-net/blob/4a63bdef269573277ad4d7fad3f8aa39a7b50e0f/sdk/core/Azure.Core/src/GeoJson/GeoJsonConverter.cs#L14) is internal, so, sure!